cloning-revera.html

```diff
@@ -0,0 +1,305 @@
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="./starter-code/img/slider-2.jpg" class="d-block w-100" alt="...">
+        <div class="carousel-caption d-none d-md-block mb-5">
+          <h5 class="display-5 mb-5">FALAURIS SOLLICITUDIN QUIS DOLOR VENENATIS FACILISIS</h5>
+          <p>Donec sed dolor eu augue dignissim pellentesque. Fusce semper tortor ornare, luctus sem in, dictum leo. Ut at congue quam! Maecenas luctus interdum odio, eget volutpat velit hendrerit eu? Suspendisse at magna ut sem blandit hendrerit non ut magna. Cras quis porta dolor. Aliquam erat volutpat. Phasellus fringilla feugiat libero nec volutpat. Maecenas fermentum bibendum […]</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="./starter-code/img/slider-3.jpg" class="d-block w-100" alt="...">
+        <div class="carousel-caption d-none d-md-block mb-5">
+          <h5 class="display-5 mb-5">SUSPENDISSE ORNARE, FELIS FEUGIAT SUSCIPIT PHARETRA</h5>
+          <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="./starter-code/img/slider-4.jpg" class="d-block w-100" alt="...">
+        <div class="carousel-caption d-none d-md-block mb-5">
+          <h5 class="display-5 mb-5">FUSCE SCELERISQUE NIBH AC NISL LUCTUS INTERDUM.</h5>
+          <p>Nam luctus velit ultricies vestibulum eleifend. Proin imperdiet varius rutrum. Curabitur placerat lacus eu ligula pellentesque euismod? Aliquam volutpat augue id nibh auctor, at varius dui semper? Suspendisse non pellentesque sapien, et sagittis erat. Quisque et dignissim elit. Sed sit amet sapien ac quam vestibulum suscipit ac id diam. Morbi accumsan est vitae tellus vulputate […]</p>
+        </div>
+      </div>
+    </div>
+    <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="visually-hidden ">Previous</span>
+    </button>
+    <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Next</span>
+    </button>
+  </div>
+
+
+
+
+  <div class="container-fluid bg-danger  ">
```

Comment:
Probably you had difficulties to align the columns here, because the classes on the Bootstrap are in the wrond position.
Look you have in the same div a class (eg. on line 93) you have`row` and a class `col`, if you check Bootrap documentation the structure to create columns is:
```
<div class="container">
  <div class="row">
    <div class="col">
      One of three columns
    </div>
    <div class="col">
      One of three columns
    </div>
    <div class="col">
      One of three columns
    </div>
  </div>
</div>
```